Kitodo/src/test/java/org/kitodo/production/services/command/CommandServiceTest.java

```diff
@@ -158,7 +158,7 @@ public void runNotExistingScriptAsync() throws InterruptedException {
         String commandString = scriptPath + "not_existing_script" + scriptExtension;
         CommandService service = new CommandService();
         service.runCommandAsync(commandString);
-        Thread.sleep(1000); // wait for async thread to finish;
+        Thread.sleep(2000); // wait for async thread to finish;
```

Comment:
Why this and the lines below are changed from 1 to 2 seconds? Is this really needed?

Reply:
Yes, my development machine is sometimes pretty slow and then the build aborts because of an error here. Maybe this should be handled completely differently than just waiting an arbitrary amount of time, but that would be something for a separate branch.

Reply:
I understand. But this will increase the build and test time for everyone. But fine for now.